CHANGELOG.md

```diff
@@ -107,7 +122,8 @@ to update to at least [cumulus-ecs-task:2.1.0](https://hub.docker.com/layers/cum
     support `aws-sdk` v3 changes.
 
 ### Added
-
+- **CUMULUS-3742**
```

Comment:
This is in the wrong section and also needs to be fixed on `master`. It should be moved up to Unreleased on both.

Reply:
Hmm good catch. This is mildly concerning. The CL entry and some of the docs below reference work from 3742, which isn't part of the backport. The PR for 3385 included these references. I think it might make sense to pull 3742 into the backport here just for clarity.

Reply:
And that's going to be tough to reconcile. I'm going to start with a new branch and cherry-pick 3742 first, which is the order these were originally merged to main.